src/main/scala/org/youshuren/endpoint/Routes.scala

```diff
@@ -0,0 +1,60 @@
+package org.youshuren.endpoint
+
+import akka.event.Logging
+import akka.http.scaladsl.server.Directives._
+import akka.http.scaladsl.server.Route
+import org.youshuren.database.Database
+import org.youshuren.model.Book
+import org.youshuren.codec.JsonFormats._
+import Implicits._
+import scala.util.{Failure, Success}
+
+object Routes {
+
+  val Books: String = "books"
+  val OnlyAvailableQueryParam: Symbol = 'only_available
+  val BookNameQueryParam: Symbol = 'book_name
```

Comment:
it is better to centralize by QueryParam/Path etc...

Reply:
something like this?
object QueryParam {
    val OnlyAvailableQueryParam: Symbol = 'only_available
    val BookNameQueryParam: Symbol = 'book_name
  }

  object Path {
    val Books: String = "books"
    val Users: String = "users"
    val Id = "id"
    val Name = "name"
    val Owner = "owner"
  }

Reply:
👍